### PR TITLE
Add a hosting options page

### PR DESCRIPTION
**Pull request by mindjuice**
Here is the first pass with placeholder data.

All feedback is welcome.

### CODE REVIEW

File: preorder/package.json [hunk 1]
```diff
@@ -25,5 +25,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "proxy": "http://localhost:8080"
```
Comment (avahowell) [inline preorder/package.json:29]: Hmm. I'm not sure I understand why this is here. The API listens on `localhost:8080`, but we don't necessarily want to proxy all requests through the API. Could you explain the rationale for adding this? Thanks!

File: hosting.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+										<li>Private locked rack: $35/month</li>
+										<li>Miner cleaning/dusting: $20/request</li>
+									</ul>
+								</li>
+							</ul>
+						</td>
+					</tr>
+				</tbody>		
+			</table>
+			<p>If you	know of a good hosting provider and you would like us to investigate adding an
```
Comment (avahowell) [inline hosting.html]: The spacing here is a bit odd (between `you` and `know`)

File: hosting.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+					listed providers.</p>
+				</div>
+				<div class="separators col-md-12">
+					<div class="red-separator"></div>
+					<div class="separator"></div>
+				</div>
+			</div>
+		</div>
+		<div class="footer">
+			<ul class="terms">
```
Comment (avahowell) [inline hosting.html]: Indentation is off here

File: hosting.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+			<li><a href="https://twitter.com/ObeliskTechHQ" target="_blank">Twitter</a></li>
+			<li><a href="assets/img/terms.pdf" target="_blank">Terms</a></li>
+			<li><a href="assets/img/privacypolicy.pdf" target="_blank">Privacy</a></li>
+		</ul>
+		<img src="assets/img/footerlogo.png" />
+		<div class="poweredby">powered by Nebulous</div>
+	</div>
+	</div>
```
Comment (avahowell) [inline hosting.html]: ditto on indentation

File: hosting.html (hunk 2 above)
Comment (mindjuice) [reply hosting.html]: Looks like a tab snuck in there instead of a space.

File: hosting.html (hunk 3 above)
Comment (mindjuice) [reply hosting.html]: I'll just beautify the whole file for consistency.

File: preorder/package.json (hunk 1 above)
Comment (mindjuice) [reply preorder/package.json:29]: This is for dev only.  If the webpack dev server can't find the requested resource, it falls back to the proxy server.  This allows you to run the API server locally without getting CORS errors.  This is how I always setup my projects when there is an API server.  If you had a different setup during development, I'd love to hear about it. 

Note that if you "namespace" your API like /api/adduser /api/addorder, then you can only proxy requests under /api, but that usually doesn't make any difference.  You can read more about this setting here: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#proxying-api-requests-in-development

**Review by mindjuice — COMMENTED** [review]
Thanks for the feedback Ava!

I was going to beautify the other HTML files for consistency, but wasn't sure if Nebulous has any policy against commits that just change formatting.

**Comment by avahowell** [comment]
@mindjuice could you provide screenshots at various screen sizes, so it's easier for the team to review these changes? Thanks.

**Comment by mindjuice** [comment]
I can do that when I get home tonight, but not at the moment.

**Comment by mindjuice** [comment]
screen shot 2017-10-23 at 8 56 14 pm

**Comment by mindjuice** [comment]
screen shot 2017-10-23 at 8 55 51 pm

**Pull request closed without merging**